### PR TITLE
211 feature request add a notification indicating the number of results in the compare tab

### PR DESCRIPTION
**Pull request by Shivhare-Ayush**
## Overview

List the GitHub issues containing the issues relevant to this pull request.

issue #211 

## What Changed

Files changed: Index, Carousel, and TabNavMenu 
Added a badge to TabNavMenu and passed data from Index to Carousel to TabNavMenu

### CODE REVIEW

**Comment by AbhiramTadepalli** [comment]
@Shivhare-Ayush did you manually change these or were you on an outdated branch?

**Review by AbhiramTadepalli — CHANGES_REQUESTED** [review]
The rest of this looks great, but keep some of the tab nav logic we had in `carousel.tsx`

**Comment by TyHil** [comment]
I think when I was helping revert some changes we accidentally checked out an earlier version of `src/components/common/Carousel/carousel.tsx`. Just made a commit adding those changes back so this should be good to go.

File: src/components/common/Carousel/carousel.tsx [hunk 1]
```diff
@@ -89,16 +79,7 @@ const Carousel = ({ names, children }: CarouselProps) => {
...
-              ? children.map((child, index) => (
-                  <div
-                    key={index}
-                    className={index === currentCard ? 'block' : 'hidden'}
-                  >
-                    {child}
-                  </div>
-                ))
-              : children}
+            {Array.isArray(children) ? children[currentCard] : children}
```
[the remark this answers is not in the document]
Comment (TyHil) [reply src/components/common/Carousel/carousel.tsx]: this is what keeps the cards mounted by hiding them instead of removing them

**Review by AbhiramTadepalli — APPROVED** [review]
Great

**Pull request merged**